db/migrate/20181211181245_create_mat_view_for_dependent_repos.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMatViewForDependentRepos < ActiveRecord::Migration[5.1]
+
+  def up
+    # This can take about 45 - 60 minutes
+    execute 'create materialized view project_dependent_repositories as
```

Comment:
Not sure why travis isn't like this...

Also maybe do this as `WITH NO DATA` so that it doesn't fill the table during the deploy?